canonical-web-and-design/jaas-dashboard
# Add terminal component to model details view.

## Done

The model details view is now gated on logging in.
After logging in the details view is rendered along with the terminal component which auto-connects to the JAAS limited terminal.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/models/foo
- You should get a 'log in message'
- Open the browser console and log in
- The model details page should render
- Wait for the terminal to connect and become functional.

## Details

The terminal is a small box in the lower left corner of the page because the markup and styling needs to be adjusted. That's out of scope for this branch and will be done as part of https://github.com/canonical-web-and-design/juju-squad/issues/889

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/861
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/863
